brainstormforce/astra
# AST-3365 - Widget section need to provide font color and font size option

### Description

Added Widget design options as a default.

### Screenshots


- https://d.pr/v/5zn66E

### Types of changes





### How has this been tested?

1. Check the footer widget design option - It should be visible and working for respective blocks like the navigation menu, normal paragraph, heading

2. Check the same above case with the Addon module activated - 
A. When the Addon module is activated that time more typography options should be visible like font size font selection, and decoration option.
B. When Addon is not activated then for typography just the font size option should be visible.

3. Check the above same for the header by adding widgets.

4. After enabling the Addon module sticky header widget design option should be visible.

### Checklist:
